openmetadata-ui/src/main/resources/ui/src/pages/TeamsAndUsersPage/TeamsAndUsersPage.component.tsx

```diff
@@ -103,23 +102,41 @@ const TeamsAndUsersPage = () => {
     setactiveUserTab(data);
   };
 
-  const isIncludes = (name: string, term: string) => {
-    return toLower(name).includes(toLower(term));
+  const fetchUserCount = (isAdmin: boolean | undefined) => {
+    return new Promise<number>((resolve) => {
+      getUsers('', 0, undefined, isAdmin, false)
+        .then((res: AxiosResponse) => {
+          if (res.data) {
+            resolve(res.data.paging.total);
+          } else {
+            throw jsonData['api-error-messages']['unexpected-server-response'];
+          }
+        })
+        .catch((err: AxiosError) => {
+          const errMsg = getErrorText(
+            err,
+            jsonData['api-error-messages']['fetch-user-count-error']
+          );
+          showErrorToast(errMsg);
+          resolve(0);
+        });
+    });
   };
 
   /**
    * Make API call to fetch all the users
    */
   const fetchUsers = (
-    limit = API_RES_MAX_SIZE,
-    paging = {} as Record<string, string>
+    limit = PAGE_SIZE_BASE,
+    paging = {} as Record<string, string>,
+    isAdmin = undefined as boolean | undefined
```

Comment:
Do we need this?
We can directly pass the `undefined`.

Reply:
Yes, we need it to keep typescript happy :)
<img width="615" alt="Screen Shot 2022-07-18 at 10 53 58 PM" src="https://user-images.githubusercontent.com/1051198/179675621-4ca59983-f37f-4685-a4c5-fc5e1b18139f.png">

Reply:
i think we can use `isAdmin = false`